src/Razor/src/Microsoft.AspNetCore.Razor.LanguageServer/WorkspaceSemanticTokensRefreshNotifier.cs

```diff
@@ -37,10 +36,24 @@ public WorkspaceSemanticTokensRefreshNotifier(
 
         _disposeTokenSource = new();
 
+        _queue = new(
+            TimeSpan.FromMilliseconds(250),
+            ProcessBatchAsync,
+            _disposeTokenSource.Token);
+
         _isColoringBackground = optionsMonitor.CurrentValue.ColorBackground;
         _optionsChangeListener = optionsMonitor.OnChange(HandleOptionsChange);
     }
 
+    private ValueTask ProcessBatchAsync(CancellationToken token)
+    {
+        _clientConnection
+            .SendNotificationAsync(Methods.WorkspaceSemanticTokensRefreshName, _disposeTokenSource.Token)
+            .Forget();
+
+        return new(Task.CompletedTask);
```

Comment:
Consider just returning `default`, which represents a synchronously completed `ValueTask<T>`. There is a note about this at https://learn.microsoft.com/en-us/dotnet/api/system.threading.tasks.valuetask-1?view=net-8.0.

> An instance created with the parameterless constructor or by the default(ValueTask<TResult>) syntax (a zero-initialized structure) represents a synchronously, successfully completed operation with a result of default(TResult).